vtStorManaged.ATest/Program.cs

```diff
@@ -29,19 +29,30 @@ static void Main(string[] args)
         {
             uint driveCount;
             eErrorCode errorCode;
+            string path = System.AppDomain.CurrentDomain.BaseDirectory;
+            ILoadRunTimeDll vtStorDll = new cLoadRunTimeDll("vtStor.dll");
+            ILoadRunTimeDll vtStorAtaDll = new cLoadRunTimeDll("vtStorAta.dll");
+            ILoadRunTimeDll vtStorScsiDll = new cLoadRunTimeDll("vtStorScsi.dll");
+            ILoadRunTimeDll vtStorAtaProtocolDll = new cLoadRunTimeDll("vtStorAtaProtocol.dll");
+            ILoadRunTimeDll vtStorScsiProtocolDll = new cLoadRunTimeDll("vtStorScsiProtocol.dll");
+            IRunTimeDll vtStorModule = vtStorDll.Load();
+            IRunTimeDll vtStorAtaModule = vtStorAtaDll.Load();
+            IRunTimeDll vtStorScsiModule = vtStorScsiDll.Load();
+            IRunTimeDll vtStorAtaProtocolModule = vtStorAtaProtocolDll.Load();
+            IRunTimeDll vtStorScsiProtocolModule = vtStorScsiProtocolDll.Load();
 
-            cDriveManagerInterface driveManager = new cDriveManagerInterface();
+            cDriveManagerInterface driveManager = new cDriveManagerInterface(vtStorModule);
 
             // Register drive enumerators
-            driveManager.RegisterDriveEnumerator(new cDriveEnumeratorAta());
-            driveManager.RegisterDriveEnumerator(new cDriveEnumeratorScsi());
+            driveManager.RegisterDriveEnumerator(new cDriveEnumeratorAta(vtStorAtaModule));
```

Comment:
Should we just define a static method for each class and it initializes the static module reference?
That way, the code to setup and identifies class-module is in centralized location and also it's less verbose everywhere else.